src/sentry/api/endpoints/user_subscriptions.py

```diff
@@ -26,7 +26,7 @@ class NewsletterValidator(serializers.Serializer):
 
 @control_silo_endpoint
 class UserSubscriptionsEndpoint(UserEndpoint):
-    owner = ApiOwner.ISSUES
+    owner = ApiOwner.ALERTS_NOTIFICATIONS
```

Comment:
not sure if this should be us or billing? i'm a little overloaded on the term `subscriptions` 😅

Reply:
yeah no, it's the emails people get. It's more like marketing but there's no eng team supporting it. I'll add it to unowned.